docs/topics/settings.rst

```diff
@@ -711,6 +711,20 @@ Scrapy  (if MEMUSAGE_ENABLED is True). If zero, no check will be performed.
 
 See :ref:`topics-extensions-ref-memusage`.
 
+.. setting:: MEMUSAGE_CHECK_INTERVAL_SECONDS
+
+MEMUSAGE_CHECK_INTERVAL_SECONDS
+-------------------------------
+
+Default: ``60.0``
+
+Scope: ``scrapy.extensions.memusage``
+
+The frequence which the current memory usage will be checked against the 
```

Comment:
Ah, now I understand. And there was a typo as well, which you gracefully ignored.  Thanks. :)  Will push a change very soon.